### PR TITLE
feat: MainPage container

### PR DESCRIPTION
**Pull request by braianj**
Screenshot 2024-05-17 at 2 51 40 PM
Screenshot 2024-05-17 at 2 51 48 PM
Screenshot 2024-05-17 at 2 52 26 PM
Screenshot 2024-05-17 at 2 58 54 PM

### CODE REVIEW

File: src/components/Routes/Routes.tsx [hunk 1]
```diff
@@ -2,29 +2,50 @@ import { Redirect, Route, Switch } from 'react-router-dom'
...
+      <>
+        <Switch>
+          <Route exact path={locations.root()} component={HomePage} />
+          <Redirect to={locations.root()} />
+        </Switch>
+        {APP_ID ? <Intercom appId={APP_ID} settings={{ alignment: 'right' }} /> : null}
+      </>
+    )
+  }
+  console.log('Routes > window.location.pathname > ', window.location.pathname)
```
[the remark this answers is not in the document]
Comment (meelrossi) [reply src/components/Routes/Routes.tsx:43]: is this working ok when user sets an invalid url? are we redirecting them to the home?

File: src/components/ConfirmPage/ConfirmPage.container.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { connect } from 'react-redux'
+import { isConnecting } from 'decentraland-dapps/dist/modules/wallet/selectors'
+import { RootState } from '../../modules/reducer'
+import { validateSubscriptionEmailRequest } from '../../modules/subscription/actions'
+import HomePage from './ConfirmPage'
```
Comment (cyaiox) [inline src/components/ConfirmPage/ConfirmPage.container.ts:5]: do you mind renaming this import?

```suggestion
import ConfirmPage from './ConfirmPage'
```

File: src/components/SignInPage/SignInPage.tsx [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import React, { useEffect } from 'react'
+// import { useHref, useParams } from 'react-router-dom'
```
Comment (cyaiox) [inline src/components/SignInPage/SignInPage.tsx:2]:
```suggestion
```

File: src/components/Notifications/Notifications.tsx [hunk 4]
```diff
@@ -14,7 +15,7 @@ export const NOTIFICATION_DESCRIPTION_TEST_ID = 'notification-description-test-i
 
 export default function Notifications(props: Props) {
   const { onGetSubscription } = props
-
+  const isTabletOrBelow = useMediaQuery('(max-width:991px)')
```
[the remark this answers is not in the document]
Comment (braianj) [reply src/components/Notifications/Notifications.tsx:18]: yes, we should

File: src/components/MainPage/MainPage.styled.ts [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+export const PageContainer = styled('div')({
+  backgroundImage: `url(${background})`,
+  backgroundPosition: 'center',
+  backgroundSize: 'cover',
+  width: '100vw',
+  minHeight: '100vh',
+  position: 'relative'
+})
+
+export const Box = styled(BoxMui)({
```
[the remark this answers is not in the document]
Comment (braianj) [reply src/components/MainPage/MainPage.styled.ts:14]: when the UI2 is ready, we will no longer need the customization

File: src/components/ConfirmPage/ConfirmPage.types.tsx [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+export type Props = {
+  onValidateSubscriptionEmailRequest: typeof validateSubscriptionEmailRequest
+  isLoading: boolean
+  address: string
+}
+
+export type MapStateProps = Pick<Props, 'isLoading' | 'address'>
+export type MapDispatchProps = Pick<Props, 'onValidateSubscriptionEmailRequest'>
+export type MapDispatch = Dispatch
```
Comment (cyaiox) [inline src/components/ConfirmPage/ConfirmPage.types.tsx:12]:
```suggestion
export type MapDispatch = Dispatch<ValidateSubscriptionEmailRequestAction>
```

File: src/components/MainPage/MainPage.types.tsx [hunk 7]
```diff
@@ -0,0 +1,9 @@
+import { Dispatch } from 'redux'
+
+export type Props = {
+  isLoading: boolean
```
Comment (cyaiox) [inline src/components/MainPage/MainPage.types.tsx:4]:
```suggestion
  isLoading: boolean
  onGetNotificationSetting: typeof getSubscriptionsRequest
```

File: src/components/MainPage/MainPage.types.tsx [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+}
+
+export type MapStateProps = Pick<Props, 'isLoading'>
+export type MapDispatchProps = {}
```
Comment (cyaiox) [inline src/components/MainPage/MainPage.types.tsx:8]:
```suggestion
export type MapDispatchProps = Pick<Props, 'onGetNotificationSetting'>
```

File: src/components/MainPage/MainPage.types.tsx [hunk 9]
```diff
@@ -0,0 +1,9 @@
...
+export type MapDispatch = Dispatch
```
Comment (cyaiox) [inline src/components/MainPage/MainPage.types.tsx:9]:
```suggestion
export type MapDispatch = Dispatch<GetSubscriptionsRequestAction>
```

**Review by cyaiox — APPROVED** [review]

**Pull request merged**